tests/compliance_operator_test.go

```diff
@@ -171,30 +175,32 @@ func TestDeleteAndAddRule(t *testing.T) {
 	err = ruleClient.Delete(context.Background(), envVarRule, metav1.DeleteOptions{})
 	assert.NoError(t, err)
 
-	time.Sleep(defaultWaitTime)
+	time.Sleep(defaultSleepTime)
```

Comment:
Do we need to sleep if we have `Eventually` in the next line?

Reply:
It's not necessary, but the first assertions will always fail if we do not sleep for a bit. By sleeping we avoid some API calls and assertions. Not opposed to removing the sleep though if you think it would be cleaner.